src/types.d.ts

```diff
@@ -39,9 +59,13 @@ interface Window {
 				framework: 'tcfv2';
 			};
 			events?: {
-				onConsentReady?: () => void;
-				onMessageReady?: () => void;
+				onConsentReady: () => void;
+				onMessageReady: () => void;
 				onMessageReceiveData?: (data: { messageId: 0 | string }) => void;
```

Comment:
Is this one optional? It isn't in `window._sp_ccpa` above.
```suggestion
				onMessageReceiveData: (data: { messageId: 0 | string }) => void;
```

Reply:
ooh no, nice one